docs/reference/cli-reference/cli-ref-sources.md

```diff
@@ -69,12 +69,18 @@ where `<operation>` is one of *List, Add, Remove, Enable, Disable,* or *Update*,
 
   Comma-separated list of valid authentication types for this source. By default, all authentication types are valid. Example: `basic,negotiate`.
 
+- **`-ProtocolVersion`**
+
+  The NuGet server protocol version to be used. Currently supported versions are `2` and `3`. Defaults to `2` if not specified. See [NuGet Server API](../../api/overview.md) for information about the version 3 protocol.
```

Comment:
Do we need a clarification that if the source ends with `index.json`, it'll be v3 automatically? 

IMO we should say, default is automatic infererence, which is 3 if the source url ends with `.json`, 2 otherwise.

Reply:
I think that's also confusing, depending on how you interpret it (trying to put myself in the mind of someone who knows almost nothing about nuget). If you do `dotnet nuget update source mySource -s https://contoso.com/nuget/v3/index.json`, NuGet does not put `protocolVersion="3"` in the nuget.config.

Also, if you explicitly write `protocolVersion="2"` in the nuget.config, but the URL ends in `/index.json`, then NuGet's going to use Version 3 anyway, which I think is a bug, but would require Client SDK API breaking changes to fix.

```suggestion
  The NuGet server protocol version to be used.
  See [NuGet.Config's packageSources documentation](../nuget-config-file.md#packagesources) for more information.
```

I'm not 100% sure I got the file path correct. Normally when writing docs, I use VSCode's Markdown extension to auto-complete links for me.